pom.xml

```diff
@@ -23,7 +23,7 @@
 		<version.com.google.apis.google-api-services-analytics>v3-rev136-1.22.0</version.com.google.apis.google-api-services-analytics>
 		<version.com.google.code.gson.gson>2.8.0</version.com.google.code.gson.gson>
 		<version.com.google.guava>21.0</version.com.google.guava>
-		<version.com.googlecode.owasp-java-html-sanitizer>20160924.1</version.com.googlecode.owasp-java-html-sanitizer>
```

Comment:
@joaofreitas-qubit I would remove this dependency from this project and let it come via transitivite from bennu. When we start having the oss 3rd party DM then we can extract this module into that DM.